client/client-multi/client-objective-c/Kaa/KaaClientFactory.h

```diff
@@ -0,0 +1,44 @@
+/**
```

Comment:
This license header was auto-generated by maven plug-in. All the files in project have license beginning with /**